v13: FAIR signposting level 1 (config flag & nginx config) (#758)
inveniosoftware/docs-invenio-rdm: docs/releases/v13/version-v13.0.0.md

Explain and screenshot of the diff tool in the admin panel

### FAIR signposting

[FAIR signposting level 2](https://signposting.org/FAIR/#level2) is enabled by default. The response header of each record's landing page includes a `Link` HTTP response header pointing to a JSON-based linkset which contains the FAIR signposting information.

In order to increase discoverability, [FAIR signposting level 1](https://signposting.org/FAIR/#level1) can be enabled with the configuration flag `APP_RDM_RECORD_LANDING_PAGE_FAIR_SIGNPOSTING_LEVEL_1_ENABLED = True`. Once enabled, FAIR signposting information will be directly included in the `Link` HTTP response header.

> Please note that for records having many authors, files, or licenses, FAIR signposting will fall back to level 2 only, in order to avoid generating excessively big HTTP response headers.

Reviewer: Is this configurable? If so, would link the config here. If not, are there parameters that an instance operator could be on the lookout for that would sway that behavior? Would be good info to share.

Author: It's currently not configurable, the change is discussed here: inveniosoftware/invenio-rdm-records#1966

Reviewer: We will probably put more of that info in a dedicated section in the future anyway. Good for now.

> However, since enabling FAIR signposting level 1 does increase the size of HTTP response headers, it is recommended to edit the `nginx` configuration and specifying [`uwsgi_buffers`](https://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_buffers) and [`uwsgi_buffer_size`](https://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_buffer_size) with higher limits than the default values, for instance:
> 
> ```nginx
> server {
>     # ...
>     # Allow for larger HTTP response headers for FAIR signposting level 1 support
>     uwsgi_buffers 8 16k;
>     uwsgi_buffer_size 16k;
>     # ...
> }
> ```

Reviewer: Since uwsgi_buffering is [off by default](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/docker/nginx/conf.d/default.conf#L66), my understanding is that `uwsgi_buffers` wouldn't matter. `uwsgi_buffer_size` is the main one that would matter by default (it's used even if `uwsgi_buffering off;`). We could indicate that with the following:

 ```suggestion
However, since enabling FAIR signposting level 1 does increase the size of HTTP response headers, it is recommended to edit the `nginx` configuration and specify [`uwsgi_buffer_size`](https://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_buffer_size) with a higher limit than the default values. If you have enabled `uwsgi_buffering on;`, then [`uwsgi_buffers`](https://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_buffers) may also be adjusted. 

```nginx
server {
    # ...
    # Allow for larger HTTP response headers for FAIR signposting level 1 support
    uwsgi_buffer_size 16k;
    # optional if uwsgi_buffering on;
    uwsgi_buffers 8 16k;
    
    # ...
}
```
```
